test/integ_tests/test_create_quantum_job.py

```diff
@@ -52,15 +52,15 @@ def test_failed_quantum_job(aws_session, capsys):
         hyperparameters={"test_case": "failed"},
     )
 
-    job_name = job.name
-    pattern = f"^arn:aws:braket:{aws_session.region}:\\d12:job/{job_name}$"
-    re.match(pattern=pattern, string=job.arn)
+    pattern = f"^arn:aws:braket:{aws_session.region}:\\d{{12}}:job/[a-z0-9-]+$"
```

Comment:
is job name lowercase only?

Reply:
As a follow up, we should put length constraints on the pattern. The UUID will be fixed in length.

Reply:
also, it seems the previous regex match for job_name - why don't we do that anymore?

Reply:
+1 to length constraint. @guomanmin exactly, the arn format is being updated to use a uuid instead of a job name, hence this change

Reply:
and I believe the uuid is lowercase, digit, and `-`, but waiting on final confirmation

Reply:
thanks for the clarfication!

Reply:
UUIDs should be case independent but it is okay if we want to enforce common practice of lowercase.